### PR TITLE
Update survey.js

### PR DESCRIPTION
**Pull request by DarkNightSkies**

### CODE REVIEW

File: spacialgaze-plugins/survey.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ class Survey {
 		let icon = '<span style="border:1px solid #' + (ended ? '777;color:#555' : '6A6;color:#484') + ';border-radius:4px;padding:0 3px"><i class="fa fa-bar-chart"></i> ' + (ended ? "Survey ended" : "Survey") + '</span>';
 		let output = '<div class="infobox"><p style="margin: 2px 0 5px 0">' + icon + ' <strong style="font-size:11pt">' + (this.allowHTML ? this.question : Chat.escapeHTML(this.question)) + '</strong></p>';
 		for (let i in this.repliers) {
-			if (this.repliers[i]) output += '<div><b>' + i + ': </b><i>"' + this.repliers[i] + '"</i><div><br/>';
+			if (this.repliers[i]) output += '<div><b>' + SG.nameColor(i) + ': </b><i>"' + this.repliers[i] + '"</i><div><br/>';
```
Comment (HoeenCoder) [inline spacialgaze-plugins/survey.js]: This should be changed from 
```javascript
'<div><b>' + SG.nameColor(i) + ': </b>' //ect
// To:
'<div>' + SG.nameColor(i, true) + ':' //ect

**Comment by DarkNightSkies** [comment]
Okay there fixed it also just figured out what `SG.nameColor(user, true)` does LOL

**Comment by HoeenCoder** [comment]
@DarkNightSkies whats your PS username?

**Comment by DarkNightSkies** [comment]
Same as here DarkNightSkies

**Comment by HoeenCoder** [comment]
Alright

**Pull request merged**